src/components/Button/ButtonStyles.scss

```diff
@@ -20,8 +20,8 @@
     text-align: center;
     padding: 0;
     margin: 0;
+    margin-bottom: -3px;
```

Comment:
The subheading font has a slight gap underneath which makes it look uncentered in the button